cmd/registry-disk-v1alpha/Dockerfile

```diff
@@ -16,15 +16,13 @@
 # Copyright 2017 Red Hat, Inc.
 #
 
-FROM debian:sid
+FROM fedora:28
```

Comment:
About image size, a user will anyway download `fedora:28` image, because it used by all other our containers, so I think it will not be a problem and we can change it when we will decide to move to `fedora-distroless` :smile:

Reply:
Well the point is, we are providing the base for registry disk. Therefore everyone will be using the same base image. Change only occurs when we change the base image for all our images. 
This can and will happen one fedora 28 is deprecated.

What we can do is base all of our images (currently based on fedora:28) on the same image, e.g. kubevirt/imagebase which we will have under control.

Reply:
ok, let's  take it. Waiting for CI.